### DISCUSSION TITLE
How to format online meetings in Sheets

### DISCUSSION THREAD

**Discussion by Eric-Anderderson** [code4recovery/12-step-meeting-list#1262] [Q&A] [answered]
We have meetings that are listed as online only. In our sheets it requests and specific address and we are getting error code during import saying ERROR. The listing online shows the address as INACTIVE (see screenshot)

How should get format the data in the cell to represent online only meetings? (see screenshot)
Here is a link to view our current G Sheet template:https://docs.google.com/spreadsheets/d/1otk733FYwzF7JA4NlXfgEy6E1JKAkWOAxabjSOuuDIE/edit?usp=sharing

![Lee-County-Area-Intergoup-Meeting-Sheet-Google-Sheets-ONLINE MEETING SCREENSHOT](https://github.com/code4recovery/sheets/assets/150064201/10c1ee4f-17ef-415c-b97d-ade6cb369c39)
![Meetings-with-‘Bonita-Springs-Happy-Hour-Zoom’](https://github.com/code4recovery/sheets/assets/150064201/f7de213c-a344-4125-a10e-5d700bd63273)

**Comment by joshreisner** [comment]
hi @Eric-Anderderson i've moved this to https://github.com/code4recovery/12-step-meeting-list/discussions - please ask new questions here going forward, thanks!

you need two things to make online meetings show up: an address that has a geographic location (looks like you have this) and a column called Conference URL (looks like you don't have this). you currently have a column called "Phone" with a bunch of URLs in there, try renaming this to Conference URL.

**Reply by joshreisner** [reply]
@Eric-Anderderson the sheets repository is for the sheets.code4recovery.org project, but since you're not using that anymore and instead importing your sheet directly to 12 Step Meeting List, this is the right place going forward

sorry to hear that did not work, did you refresh your feed on the import & settings page? when i import it to our demo site these meetings have zoom links now: https://code4recovery.org/tsml-ui?search=bonita+zoom

**Reply by joshreisner** [reply, marked as answer]
also @Eric-Anderderson i'm noticing your timezone is set to Denver, please set this to New York at Wordpress > Settings. otherwise this will throw the times off for your meetings

**Comment by Eric-Anderderson** [comment]
Thank you for moving it. Thought it belonged with Sheets.

I renamed (columns) Phone to Conference URL and refreshed and this did not solve the problem, unfortunately.

Any other suggestions? Please and thank you.

**Comment by Eric-Anderderson** [comment]
I deleted the prior feed and reconnected it and it's correctly working now.
Also updated the Time Zone. Thank you, I had no idea. Appreciate your help.

On Mon, Nov 13, 2023 at 4:01 PM Josh Reisner ***@***.***>
wrote:

> also @Eric-Anderderson <https://github.com/Eric-Anderderson> i'm noticing
> your timezone is set to Denver, please set this to New York at Wordpress >
> Settings. otherwise this will throw the times off for your meetings
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/1262#discussioncomment-7558579>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDY4YSOQLQHGRWPXKPB2RBTYEKDCJAVCNFSM6AAAAAA7JT2CQ6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TKNJYGU3TS>
> .
> You are receiving this because you were mentioned.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/1262/comments/7558579
> @github.com>
>


-- 
*ERIC ANDERSON I CEO/Founder*
*Rapture Creative*
*239.398.1009*
*-------------------------*
How'd we do?
<https://rapture-creative.pdqs.mobi/rapture-creative/review/add/?stage=share>
<http://rapture-creative.pdqs.mobi/rapture-creative/review/add/?stage=share>
<http://instagram.com/rapturecreative/>
<http://facebook.com/RaptureCreative/>
<https://www.linkedin.com/company/rapture-creative>
<http://youtube.com/c/RaptureCreativeLLC>
